### DISCUSSION TITLE
general chat

### DISCUSSION THREAD

**Discussion by IzaacCoding36** [IzaacCoding36/IzaacWeb#7] [General]
Wanna talk to other people or even about my website? No worries, in this general chat, you can say whatever you want, but be nice, because I don't want anyone beign reported or anything like that.

Enjoy

**Comment by CodeRaptor92** [comment]
Hello! if you need any help with your little project you can call me to review it, ok? >_<

nothing personal but as i dont know how to program i can only review your code to look out for errors or bugs. (that's all i can do :| )

**Reply by IzaacCoding36** [reply]
Thanks! Any help is appreciated
